denops/ddu/ddu.ts

```diff
@@ -1403,6 +1403,8 @@ export class Ddu {
     for (const alias of aliases) {
       add(alias);
     }
+
+    return;
```

Comment:
Why the return is needed?

Reply:
This may not be needed actually. Just trying to make sure that the resolution of Promise happened in the end of the function.

Reply:
Please remove the line.